src/EditorControls.js

```diff
@@ -239,6 +239,24 @@ class EditorControls extends Component {
         }
         break;
 
+      case EDITOR_ACTIONS.DELETE_TRANSFORM:
+        if (isNumeric(payload.transformIndex)) {
+          for (let i = 0; i < graphDiv.data.length; i++) {
+            if ((graphDiv.data[i].uid === payload.traceUid) !== -1) {
+              graphDiv.data[i].transforms.splice(payload.transformIndex, 1);
```

Comment:
!== -1
....
I don't get it

Reply:
whoops, nice catch, that's me refactoring badly